package.json

```diff
@@ -17,6 +17,7 @@
     "memory-fs": "~0.4.1",
     "mkdirp": "~0.5.0",
     "node-libs-browser": "^2.0.0",
+    "sinon": "^1.17.7",
```

Comment:
@wtgtybhertgeghgtwtg Good spot - it should be in `devDependencies`. I've put up a PR to correct; #3707